sass/_jetpack.scss

```diff
@@ -60,90 +60,98 @@ Related Posts
-            margin-right: 20px;
-            transition: all .2s ease-in-out;
-        }
-    }
-    
-    li {
-        border-bottom: 1px solid #666;
-        margin-bottom: 20px;
-        padding-bottom: 20px;
-        
-        &:hover a.bump-view {
-            color: #aaa;
-
-            .widgets-list-layout-blavatar {
-                opacity: .8;
-            }
-        }
-    }
-    
-    /*
-     * I would like nothing better than to use this instead:
-     *
-     * &-links {
-     *
-     * Unfortunately, that results in the same level of specificity as defined by Jetpack's styles,
-     * which, because the plugin's CSS is loaded after Cover's, overrides it.
-     * As a result, we have to increase the specificity if we don't want to resort to using !important everywhere.
-     */
-    .widgets-list-layout-links {
-        float: none;
-        width: auto;
-    }
+	a.bump-view {
+		border: 0;
+		padding: 0;
+
+		.widgets-list-layout-blavatar {
+			margin-right: 20px;
+			transition: all .2s ease-in-out;
+		}
+	}
+
+	li {
+		border-bottom: 1px solid #666;
+		margin-bottom: 20px;
+		padding-bottom: 20px;
+
+		&:hover a.bump-view {
+			color: #aaa;
+
+			.widgets-list-layout-blavatar {
+				opacity: .8;
+			}
+		}
+	}
+
+	/*
+	* I would like nothing better than to use this instead:
+	*
+	* &-links {
+	*
+	* Unfortunately, that results in the same level of specificity as defined by Jetpack's styles,
+	* which, because the plugin's CSS is loaded after Cover's, overrides it.
+	* As a result, we have to increase the specificity if we don't want to resort to using !important everywhere.
+	*/
+	.widgets-list-layout-links {
+		float: none;
+		width: auto;
+	}
 }
 
 /*--------------------------------------------------------------
 Image Grid
 --------------------------------------------------------------*/
 .widget-grid-view-image {
-    width: 33.33%;
-    
-    @media screen and (max-width: $width-phone) {
-        width: 50%;
-    }
-
-    a.bump-view,
-    &:nth-child(even) a.bump-view {
-        border: 0;
-        margin: 0;
-        opacity: .8;
-
-        &:hover {
-            opacity: 1;
-        }
-    }
-    
-    a[title] {
-        position: relative;
-        
-        &:after {
-            background-image: linear-gradient(rgba(0, 0, 0, 0), rgba(0, 0, 0, .5));
-            bottom: 0;
-            color: #fff;
-            content: attr(title);
-            padding: 10px;
-            position: absolute;
-            width: 100%;
-            @include rem(font-size, 14px);
-        }
-    }
-
-    img {
-        display: block;
-        width: 100%;
-    }
+	width: 33.33%;
+
+	@media screen and (max-width: $width-phone) {
+		width: 50%;
+	}
+
+	a.bump-view,
+	&:nth-child(even) a.bump-view {
+		border: 0;
+		margin: 0;
+		opacity: .8;
+
+		&:hover {
+			opacity: 1;
+		}
+	}
+
+	a[title] {
+		position: relative;
+
+		&:after {
+			background-image: linear-gradient(rgba(0, 0, 0, 0), rgba(0, 0, 0, .5));
```

Comment:
Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets